src/RestSharp/Parameters/Parameter.cs

```diff
@@ -34,7 +34,7 @@ public static Parameter CreateParameter(string? name, object? value, ParameterTy
             ParameterType.GetOrPost   => new GetOrPostParameter(name!, value?.ToString(), encode),
             ParameterType.UrlSegment  => new UrlSegmentParameter(name!, value?.ToString()!, encode),
             ParameterType.HttpHeader  => new HeaderParameter(name, value?.ToString()),
-            ParameterType.RequestBody => new BodyParameter(name, value!, Serializers.ContentType.Plain),
+            ParameterType.RequestBody => new BodyParameter("", value!, string.IsNullOrEmpty(name) ? Serializers.ContentType.Plain : name!),
```

Comment:
That will break things. Unlike RS 106, the latest version allows you to add multiple body parameters. It works with multipart form data. There, parameters must have names, also body parameters.

Reply:
Yeah, not sure I like the fix either but so much code does it that way :(

Reply:
The question is: do we want to make it compatible or force people to use the right API. In the latter case, we can throw an exception if the parameter name looks like content type, and the exception would say: "it won't work, use `AddStringBody`". I can add an overload for `AddStringBody` that would accept the parameter name.

Reply:
That’s a good idea. For us it took a long time to figure out what was wrong as I had to inspect the outgoing packets to finally work out it was not sent with the right content type. A logical exception explaining what is wrong would save those folks with the same problem a lot of time.